conductr_cli/conduct_logging.py

```diff
@@ -112,3 +113,16 @@ def raise_for_status_inc_3xx(response):
     response.raise_for_status()
     if response.status_code >= 300:
         raise HTTPError(status_codes._codes[response.status_code], response=response)
+
+
+def format_timestamp(timestamp, args):
+    date = arrow.get(timestamp)
+
+    if args.date and args.utc:
+        return date.to('UTC').strftime('%Y-%m-%dT%XZ')
```

Comment:
When UTC you can be non-locale specific with the time i.e. make it a full ISO 8601 timestamp. :-)